spirv_glsl.cpp

```diff
@@ -4948,7 +4948,13 @@ string CompilerGLSL::convert_float_to_string(const SPIRConstant &c, uint32_t col
 
 			char print_buffer[32];
 			sprintf(print_buffer, "0x%xu", c.scalar(col, row));
-			res = join(bitcast_glsl_op(out_type, in_type), "(", print_buffer, ")");
+
+			string comment = "inf";
```

Comment:
`const char *`, no need to dynamically allocate the comment here.

Reply:
Brilliant! Thanks!